internal/controller/observability/clusterlogforwarder_controller.go

```diff
@@ -50,6 +50,8 @@ func (r *ClusterLogForwarderReconciler) Reconcile(ctx context.Context, req ctrl.
 		return defaultRequeue, err
 	}
 
+	removeStaleStatuses(r.Forwarder)
```

Comment:
@jcantrill Do we need some sort of error handling here?

Reply:
This is working on an existing struct where I don't see any opportunity for an error to be produced so there is nothing to handle.  At best it would be maybe a nil pointer :man_shrugging: